Week3/src/isInputValid.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const { ErrorWithStatus } = require('./error');
+
+const USERS_REG = /^\/users$/i;
```

Comment:
nice that you added some validation 👍  As you can see this is quite cumbersome to implement. Luckily, there are alternative ways to do this using what is called a schema.  You can red more here http://json-schema.org/learn/getting-started-step-by-step.html